### PR TITLE
WT-7295 Compatibility with older versions of WT

### PR DESCRIPTION
**Pull request by raviprakashgiri29**
The WT-5886 change caused MongoDB multi-version tests to fail in windows machines. WT-5886 changes are reverted, this ticket would incorporate the WT-5886 changes with other fixes.

### CODE REVIEW

File: src/docs/build-windows.dox [hunk 1]
```diff
@@ -101,4 +101,8 @@ compression; see @ref compression for more information.
 Configure WiredTiger for <a href="http://www.zlib.net/">zlib</a>
 compression; see @ref compression for more information.
 
+@par \c --disable-standalone-build
```
Comment (kommiharibabu) [inline src/docs/build-windows.dox:104]: Can you add a similar doc change for Linux variant also?

**Comment by kommiharibabu** [comment]
@raviprakashgiri29 The PR title should point to the original title of WT-5886.

**Review by keithbostic — APPROVED** [review]
lgtm.

File: src/docs/build-windows.dox (hunk 1 above)
Comment (raviprakashgiri29) [reply src/docs/build-windows.dox:104]: Updated `build-posix.dox`.

**Comment by raviprakashgiri29** [comment]
> @raviprakashgiri29 The PR title should point to the original title of [WT-5886](https://jira.mongodb.org/browse/WT-5886).

@kommiharibabu Updated the PR title.

**Review by kommiharibabu — APPROVED** [review]
LGTM

**Pull request merged**